lighthouse-core/audits/seo/meta-description.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+
+class Description extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Content Best Practices',
+      name: 'meta-description',
+      description: 'Document has a meta description',
+      failureDescription: 'Document does not have a meta description',
+      helpText: 'Meta descriptions may be included in search results to concisely summarize ' +
+          'page content. Read more in the ' +
+          '[Search Console Help page](https://support.google.com/webmasters/answer/35624?hl=en#1).',
+      requiredArtifacts: ['Description']
```

Comment:
`Description` is rather generic, what about `MetaDescription` or start namespacing with `SEODescription` if there are going to be many more seo specific ones?